en/stacks/audio/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Audio management
+---
+
+# Audio management
+
+There are currently two different snaps that help manage audio under Ubuntu Core Stacks:
+
+1. ALSA Utils
```

Comment:
As we're talking about the snaps above we should give the right names here which are 'alsa-utils' and 'pulseaudio'.